### DISCUSSION TITLE
Option to change "Records per page" control in QTable

### DISCUSSION THREAD

**Discussion by yu0A** [quasarframework/quasar#16693] [Ideas / Proposals] [answered]
Quasar has a prop "pagination-label", but it only changes pagination label but not "Records per page:".
![image](https://github.com/quasarframework/quasar/assets/111437058/ae179ae2-17c4-4942-a52e-d4a258d5296e)

**Comment by RebwarMustafa** [comment, marked as answer]
You can use `rows-per-page-label` prop to change _Records per page_ text

**Reply by yu0A** [reply]
It works. Thanks!

**Reply by Laura0310** [reply]
Could you show me how you did it? I tried, but it didn't work for me! Thanks

**Reply by yusufkandemir** [reply]
@Laura0310 https://codepen.io/yusufkandemir/pen/xxBdygp?editors=101

**Reply by devrimmehmet** [reply]
![1](https://github.com/quasarframework/quasar/assets/29556053/b0fc3f0e-2634-4c1a-9c82-e45e35424471)

So, do you know what I should use for the 'of' expression in between?

By the way, 'rows-per-page-label' works, thank you.

**Reply by yusufkandemir** [reply]
@devrimmehmet that one is controlled by the `pagination-label` prop. But, if what you are looking for is to simply translate the text, then you can set the language pack to use. You can use the `tr` language pack. See https://quasar.dev/options/quasar-language-packs#configuring-the-default-language-pack

**Reply by devrimmehmet** [reply]
[SOLVED]Thank you very much Mr. Yusuf, this is exactly what I needed. @yusufkandemir
